src/locales/en-US.json

```diff
@@ -1061,6 +1061,9 @@
     "updateLogicalMemoryPageSetup": "Update system memory page setup",
     "updateLogicalMemorySize": "Update logical memory block size",
     "updatePageSetup": "Update system memory page setup",
+    "dynamicIoDrawerAttachmentTitle":"Dynamic I/O Drawer Attachment",
```

Comment:
Rename as "Dynamic I/O drawer attachment" to be consistent with the rest of the content in the page.